zap/src/main/java/org/zaproxy/zap/extension/api/PhpAPIGenerator.java

```diff
@@ -156,24 +177,24 @@ private void generatePhpElement(ApiElement element, String component, String typ
                 .append(API.API_KEY_PARAM)
                 .append("' => $")
                 .append(API.API_KEY_PARAM)
-                .append(")");
+                .append("]");
 
         List<ApiParameter> optionalParameters =
                 element.getParameters().stream()
                         .filter(e -> !e.isRequired())
                         .collect(Collectors.toList());
         if (!optionalParameters.isEmpty()) {
-            out.write("\t\t$params = ");
+            out.write("        $params = ");
             out.write(reqParams.toString());
             out.write(";\n");
             reqParams.replace(0, reqParams.length(), "$params");
 
             for (ApiParameter parameter : optionalParameters) {
                 String name = parameter.getName();
                 String varName = name.toLowerCase(Locale.ROOT);
-                out.write("\t\tif ($" + varName + " !== NULL) {\n");
```

Comment:
Switched tabs into spaces as that is what is being used by standards. In any case, all the code is reformatted by proper PHP linters in the PHP API client.